### PR TITLE
fix: Need to adapt to the width of mobile webpage on CustomerList

### PR DESCRIPTION
**Pull request by i53577**
JIRA: https://jira.tools.sap/browse/CXSPA-3596

### CODE REVIEW

File: feature-libs/asm/components/customer-list/customer-list.component.html [hunk 1]
```diff
@@ -294,7 +294,7 @@ <h2 id="asm-customer-list-title" class="title modal-title">
         <!-- Modal Foot -->
         <div
           class="cx-dialog-sub-header modal-header cx-dialog-foot"
-          [class.tablet-mobile]="(breakpoint$ | async) !== BREAKPOINT.md"
```
Comment (i53577) [inline feature-libs/asm/components/customer-list/customer-list.component.html:297]: `tablet-mobile` is useless.
use `asm-mobile-pagination` instead to make sure the align direction is correct.

File: feature-libs/asm/components/default-asm-pagination.config.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { PaginationConfig } from '@spartacus/storefront';
 
 export const defaultAsmPaginationConfig: PaginationConfig = {
   pagination: {
-    rangeCount: 4,
+    rangeCount: 2,
```
Comment (i53577) [inline feature-libs/asm/components/default-asm-pagination.config.ts:11]: reduce the length of the pagination component.

File: feature-libs/asm/styles/components/_customer-emulation.component.scss [hunk 3]
```diff
@@ -39,6 +39,7 @@
     }
     .cx-asm-uid {
       color: #556b82;
+      word-break:break-all;
```
Comment (i53577) [inline feature-libs/asm/styles/components/_customer-emulation.component.scss]: break the line when `uid` is too long.

File: feature-libs/asm/styles/components/_customer-emulation.component.scss [hunk 4]
```diff
@@ -39,6 +39,7 @@
     }
     .cx-asm-uid {
       color: #556b82;
+      word-break: break-all;
```
Comment (i53577) [inline feature-libs/asm/styles/components/_customer-emulation.component.scss:42]: break the line when `uid` is too long

**Review by i333055 — APPROVED** [review]

**Pull request merged**